hazelcast/src/main/java/com/hazelcast/sql/impl/expression/service/GetDdlFunction.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2008-2023, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.sql.impl.expression.service;
+
+import com.hazelcast.map.IMap;
+import com.hazelcast.nio.serialization.IdentifiedDataSerializable;
+import com.hazelcast.sql.impl.QueryException;
+import com.hazelcast.sql.impl.SqlDataSerializerHook;
+import com.hazelcast.sql.impl.expression.Expression;
+import com.hazelcast.sql.impl.expression.ExpressionEvalContext;
+import com.hazelcast.sql.impl.expression.TriExpression;
+import com.hazelcast.sql.impl.row.Row;
+import com.hazelcast.sql.impl.schema.Mapping;
+import com.hazelcast.sql.impl.schema.type.Type;
+import com.hazelcast.sql.impl.schema.view.View;
+import com.hazelcast.sql.impl.type.QueryDataType;
+
+import static com.hazelcast.jet.impl.JetServiceBackend.SQL_CATALOG_MAP_NAME;
+import static com.hazelcast.sql.impl.expression.string.StringFunctionUtils.asVarchar;
+
+public class GetDdlFunction extends TriExpression<String> implements IdentifiedDataSerializable {
+    static final String TABLE_NAMESPACE = "table";
```

Comment:
according to TDD (https://github.com/hazelcast/hazelcast/pull/23602/files) this should be `relation`, according to https://hazelcast.atlassian.net/browse/HZ-2024: `table`. Which one is correct?
For me `table` is counter-intuitive for views and user defined types, eg. `SELECT get_ddl('table', 'my_view')`

Reply:
TDD was silently updated, haven't seen these changes before you directed me, thanks.